src/crud/session.py

```
@@ -420,6 +421,36 @@ async def delete_session(
+        vector_store = get_vector_store()
+
+        if embeddings:
+            # Build vector IDs: {message_id}_{chunk_index}
+            vector_ids = [f"{e.message_id}_{e.chunk_index}" for e in embeddings]
+
+            # Try to delete from vector store (best effort)
+            try:
+                namespace = vector_store.get_message_namespace(workspace_name)
+                await vector_store.delete_many(namespace, vector_ids)
+                logger.debug(
+                    f"Deleted {len(vector_ids)} message vectors for session {session_name}"
```

Ruff (0.14.7)

Logging statement uses f-string

(G004)